index.js

```diff
@@ -0,0 +1,2 @@
+require('./draganddrop');
+module.exports = 'ang-drag-drop';
```

Comment:
I did it this way following the example at http://blog.npmjs.org/post/114584444410/using-angulars-new-improved-browserify-support, but I'm not sure if there's any purpose beyond allowing the use of require('angular-native-dragdrop') directly in the dependencies array when declaring a new angular module. I guess one extra benefit is less confusion between what module name to use (i.e. the npm name 'angular-native-dragdrop' or angular module name 'ang-drag-drop')